src/components/Navbar.tsx

```diff
@@ -26,9 +26,26 @@ export const Navbar: React.FC = () => {
 					</Link>
 				</li>
 				<li>
-					<Link href="/homeseeker">
-						<span className="text-black">Homeseeker</span>
-					</Link>
```

Comment:
This is gonna be a merge conflict but you can copy how I did it with `TutorTimeDropdown` for yours too.